src/types/options/index.ts

```diff
@@ -36,36 +36,62 @@ export type KnownOptionsPayloadFromType<T extends string> = Extract<SlackOptions
  */
 export interface BlockSuggestion extends StringIndexed {
```

Comment:
At the moment I lean against trying to consume these types, as internally in bolt-js we already have a model of `block_suggestion` which needs to be compatible with legacy and next-gen  event payloads. I don't want to be using different types for block_suggestion events because they aren't different between next-gen and today's block_suggestion (nor do we want them to drift). 

It would also add additional dependency for us on deno_slack_sdk to be selectively consuming `deno_slack_sdk` function handler types. Currently we're keeping this dependency limited to manifest generation. 

Would love to know @filmaj's thoughts on this though since sounds like this could be a candidate for the shared types work he's been prototyping.